### DISCUSSION TITLE
XP boost for new players

### DISCUSSION THREAD

**Discussion by zeeb1337** [chromiecraft/chromiecraft#7594] [Suggestions]
Essentially dynamic XP, a way for new players to catch up to the rest of the population. I think this would make CC a more attractive option to retain a more lively player base. Having the experience slowing down the closer they get to the level cap. This shouldn't hurt the capped player base either since the last 10 levels would be 1x (or whatever base XP is). This would let them become part of the general community much faster, instead of feeling they have to spend a significant portion of their playtime "alone".

For example:
- 1-9: 7x
- 10-19: 6x
- 20-29: 5x
- 30-39: 4x
- 40-49: 3x
- 50-59: 2x
- 60-70: 1x

If someone wants to specifically test the lower levels, there could be a command added that sets XP to static 1x. Or they could manually decide to do all quests etc.

More players  = more issues found = better AzerothCore.
At least that's my view on it.

**Comment by krutoj87** [comment]
Currently we have x2 exp rates and there is a command to lower them for players who want slower leveling. 
But I would agree with having higher rates aswell, for players that don't enjoy leveling. I think it would increase player numbers in the long run and I also already made a suggestion like this aswell. 
I just don't see the purpose to slow leveling down as players get closer to their goal. I don't think this would be a good game experience, if your goal moves further away as you get closer. 
So I would rather be for increased rates, that stay the same up to max level. 

However I don't really see this happening on this server. While we do get some QoL features, the server still tries to stay as close as possible to the original game.

**Reply by zeeb1337** [reply]
I mean, it would be good if it did. But not during testing, which is the whole point of the server. Unless I've misread it and is really out hiking here. So we're in agreement at least on that it should be much faster leveling since you can also choose to slow level. Rates can be discussed by the maintainers once a decision to increase the rate(s) has been made. But I personally like this approach. It works wonders on Tibia private servers. Dynamic XP stages practically being the norm on those.

I'm running around in Westfall and I can't cover as much ground as I'd like to because I just don't gain levels fast enough. I'm also, most of the time, alone in the zones I'm currently leveling in. Antother issue with a low playerbase is that dungeons aren't tested, you just don't find any RDF groups.

The server really needs a more active playerbase spread across the game. Today I found a pretty significant pathfinding issue involving spellcasters. This was a general issue with spellcasters, so it wasn't level dependant. According to other players on the server.

Why I suggested dynamic XP was so that already heavily tested content will get less priority. Players usually tend to heavily populate early game and then phase out the closer to cap you get. This was my primary reasoning.

Also, this would be a QoL feature. Just like 1v1 Arenas. So the last argument doesn't really apply. I'd say even less since a larger playerbase would be much more preferred over 1v1 Arenas. Increasing the XP would definitely attract the more casual players, which outnumber the more dedicated ones. Even if the casual players are less likely to report anything they find, they'd still most likely talk about it with other players or on World chat, making it known to everyone else.

**Reply by krutoj87** [reply]
I wouldn't necessarily conclude, that the low level content has been tested more, than the higher level content, just because it was played more. Because those people, that quit before reaching max level most likely won't submit bug reports. 

Also a lot of the bugs aren't really related to one specific quest or mob, it's often times general mob behaviour, like leashing, path finding, no diminishing returns on CC abilities and stuff like that. This exists throughout all levels and it's already reported for a while. 

With the exception of a few quests, most quests are working in my opinion. And for those that don't work, the bugs already have been reported too.

The only thing that would need continous testing is, to make sure fixes don't break other things. But even with higher rates you will always have players, that rather level slow or challenge mode players. Those will always test everything extensively and some times even have recordings of bugs, because it seems some challenge players record their gameplay, to be able to prove it was because of a bug, when they die because of a bug.
